src/procstat.c

```diff
@@ -1,7 +1,7 @@
 /*
  *   BSD LICENSE
  *
- *   Copyright (C) 2016 LightBits Labs Ltd. - All Rights Reserved
```

Comment:
It would be nice to add a bit more detailed in the commit-msg for reviewers who aren't familiar with the matter:
- what's bad about the global_lock
- the assumption we base this two-tier locking design on (that the application has a single thread/cpu ownership over every sub-directory under root)

Reply:
I planned to add the details to the merge comment.